libc/src/math/generic/asinf.cpp

```diff
@@ -109,7 +109,7 @@ LLVM_LIBC_FUNCTION(float, asinf, (float x)) {
       fputil::set_errno_if_required(EDOM);
       fputil::raise_except_if_required(FE_INVALID);
     }
-    return x + FPBits::build_nan(Sign::POS, FPBits::FRACTION_MASK).get_val();
+    return x + FPBits::build_nan().get_val();
```

Comment:
@lntue independent from this patch, I don't really understand why we need to add `x` and `NaN`. Is there a reason?

Reply:
It was carried from the old implementations, where we didn't have `build_quiet_nan` yet.  They should all be replaced by `build_quiet_nan`.